src/quacc/recipes/espresso/_base.py

```diff
@@ -76,9 +83,13 @@ def run_and_summarize(
         parallel_info=parallel_info,
     )
 
+    updated_copy_files = _prepare_copy(
+        copy_files, atoms.calc._user_calc_params, atoms.calc.template.binary
+    )
```

Comment:
The function takes keyword arguments, so we should specify that rather than supplying them as keyword-less positional arguments.